rclc_lifecycle/README.md

```diff
@@ -86,4 +76,4 @@ An example, how to use the RCLC Lifecycle Node is given in the file `lifecycle_n
 
 ## Limitations
```

Comment:
I am fine, if you just highlight here the functions for rcl_lifecycle_node_fini. 
For a complete example, call `rclc_executor_fini`, `rcl_node_fini` and `rclc_support_ini` too.

(Note: as these lines have not changed, I cannot comment on the actual line number ...)